system-images/compat/Makefile

```diff
@@ -2,10 +2,17 @@
 
 .DELETE_ON_ERROR:
 
-all: compat.tar
+all: docker-images/aarch64.tar
 
-compat.tar: Dockerfile target/aarch64-unknown-linux-musl/release/compat
-	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --tag start9/x_system/compat --platform=linux/arm64 -o type=docker,dest=compat.tar .
+clean:
```

Comment:
:speech_balloon: Not sure if this build things will work with both, but I would rather verify than just reason on the code. I'm assuming that these work, if they don't it will be a fail fast, so little harm